model.js

```diff
@@ -98,6 +104,23 @@ exports.updateArticleVotes = (article_id, inc_votes) => {
     });
 };
 
-    
+
+exports.removeComment = (comment_id) =>{
+  if (!comment_id){
+    return Promise.reject({ status: 400, msg: "Bad Request" })
+
+  }
+  return db
+    .query("SELECT * FROM comments WHERE comment_id = $1;", [comment_id])
```

Comment:
This query isn't needed. you can refactor this model to make one db query and reject where needed. Look up the RETURNING * on a delete.